utils/unittest/execution_state.go

```diff
@@ -19,7 +19,7 @@ import (
 const ServiceAccountPrivateKeyHex = "e3a08ae3d0461cfed6d6f49bfc25fa899351c39d1bd21fdba8c87595b6c49bb4cc430201"
 
 // Pre-calculated state commitment with root account with the above private key
-const GenesisStateCommitmentHex = "2fac1ddd9cdf8d036e97fed716f70183ce75b944eb1fde7cc97cb0a7ebcb4959"
+const GenesisStateCommitmentHex = "d199da82127e0da6304191bb89296683cfafa65fe5cd6d7a91d743ee2e1b95d0"
```

Comment:
NOTE: These are changed on purpose, as the storage layer is slightly altered here.